_build_cfg.yml

```diff
@@ -388,7 +388,7 @@ Topics:
     Distros: openshift-*
   - Name: Setting Quotas
     File: quota
-  - Name: Setting Limit Ranges
+  - Name: Resource Limits
```

Comment:
"Setting Limit Ranges" was the new name given to this topic when it was moved via https://github.com/openshift/openshift-docs/pull/2059/files#diff-e612d75a53fc180c5c4512806a7b1c31.